assets/svg/social/instagram.svg

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="32" viewBox="0 0 32 32">
+  <g fill="#4C4D4E" fill-rule="nonzero">
```

Comment:
Why is the icon filled with a color ? I think you can can clean it a little bit ;)

Reply:
You can remove this `<g>` tag (not its content)